pandas/io/parsers/readers.py

```diff
@@ -432,7 +432,7 @@
 
 Examples
 --------
->>> pd.{func_name}('data.csv')  # doctest: +SKIP
+>>> df = pd.{func_name}('data.csv')  # doctest: +SKIP
```

Comment:
What are you trying to do here?

Reply:
I’m just trying to make the examples section here correct.

The example as stated reads the csv, loads it into a dataframe, and then immediately throws it away. No one uses read_csv this way.

Reply:
I don't think we are doing this anywhere else explicitly like that

Reply:
agreed, this is fine as-is - closing then, but thanks anyway for the suggestion